resources/install.rb

```diff
@@ -4,7 +4,7 @@
 default_action :create
 
 property :yum_baseurl, String, default: 'https://jfut.integ.jp/linux/ganeti/$releasever/$basearch'
-property :yum_gpgkey, String, default: 'https://raw.githubusercontent.com/jfut/ganeti-rpm/refs/heads/main/rpmbuild/integ-ganeti-release-10/SOURCES/RPM-GPG-KEY-integ-ganeti-10'
+property :yum_gpgkey, String, default: 'https://jfut.integ.jp/linux/ganeti/RPM-GPG-KEY-integ-ganeti'
```

Comment:
Can you please create a helper method which has EL10 using the other key but otherwise use the current key?

I created an [upstream issue](https://github.com/jfut/ganeti-rpm/issues/73) get this added on the mirror so we'll need to change this again before we merge it.